test/evp_test.c

```diff
@@ -660,6 +660,36 @@ static int cipher_test_enc(EVP_TEST *t, int enc,
         /* We supply the data all in one go */
         if (!EVP_CipherUpdate(ctx, tmp + out_misalign, &tmplen, in, in_len))
             goto err;
+    } else if (frag == 2) {
+        if (EVP_CIPHER_CTX_block_size(ctx) > 1
```

Comment:
Even though it's not critical, I'd say it would be appropriate to cache block_size value.